docs/reference/mistakecheckers.md

```diff
@@ -0,0 +1,16 @@
+## Mistake Checkers
+This refers to validating user actions against what is expected and considered correct in a lab.
+## Types of Errors
+In the simulation, there are a variety of possible things to check for, though they tend to resolve to two categories: general and lab-specific. 
+### General
```

Comment:
Correct me if I'm wrong on this, but I think it might be worth specifying that, for a General Mistake, like the example given or using a pipette without a tip or something, the LabObject itself should generally be the thing to call the LabLog functions, and for the Lab-Specific mistakes, they should generally use ReportAction()?

Reply:
You're right. I made sure to include that information in the General and Lab-Specific sections.